client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -35,6 +39,12 @@
 		padding-block-start: 24px;
 	}
 
+	.dataviews-view-table thead {
+		position: sticky;
+		top: 0;
+		z-index: 2;
+	}
+
```

Comment:
I made a slight improvement that makes the table header stay on the top. Ideally, we would want the table body to be only the scrollable content, but it is difficult to override the styling without major overrides.

Reply:
🙌